README.md

```diff
@@ -18,6 +24,10 @@ wget --no-check-certificate -P fineract-provider/gradle/wrapper https://github.c
 
 curl --insecure -L https://github.com/apache/fineract/raw/develop/fineract-provider/gradle/wrapper/gradle-wrapper.jar > fineract-provider/gradle/wrapper/gradle-wrapper.jar
 
+Instructions to download MySQL
+============
+If MySQL is not installed in the device, download from [MySQL](https://www.mysql.com/downloads/ "MySQL").
+
```

Comment:
Maybe add details of the MySQL version required like you have for Java ? Else, remove this section and add a hyperlink directly to the requirements section itself, as there isn't much information here to warrant the addition of another section.